pkg/cli/commands.go

```diff
@@ -55,6 +57,17 @@ func MakeMainCommand(
 
 		l := ctxzap.Extract(runCtx)
 
+		otelShutdown, err := uotel.InitOtel(context.Background(), v.GetString("otel-collector-endpoint"), name)
```

Comment:
Any reason this doesn't use the runCtx?

Reply:
Same answer for both cases. If `runCtx` gets cancelled, we want to be sure that otel can still flush its data before the process is shutdown.